doc/overviews/rate-limiting.md

```diff
@@ -201,31 +209,31 @@ spec:
   limits:
     "toystore-all":
       rates:
-      - limit: 5000
-        window: 1s
+        - limit: 5000
```

Comment:
Both are valid yaml docs. 

Not arguing to revert your changes, but wondering if there is a style guide that says it should be like that.

Reply:
its my linter/formatter that changed it i use default [prettier](https://prettier.io/) and [stylelint](https://stylelint.io/)